gordo/client/cli/client.py

```diff
@@ -0,0 +1,232 @@
+"""Gordo-client click command."""
+import json
+import os
+import pickle  # noqa:S403
+import sys
+from datetime import datetime
+from pprint import pprint
+from typing import Iterable, List, Optional, Tuple, Union
+from typing.io import IO
+
+import click
+import pandas as pd
+import simplejson
+import yaml
+from gordo_dataset.data_provider import providers
+from pkg_resources import get_distribution
+from requests import Session
+
+from gordo.client import Client
+from gordo.client.cli.custom_types import DataProviderParam, IsoFormatDateTime, key_value_par
+from gordo.client.forwarders import ForwardPredictionsIntoInflux
+
+
+@click.group("client")
+@click.version_option(version=get_distribution("gordo.client").version, prog_name="gordo-client")
+@click.option("--project", help="The project to target")
+@click.option("--host", help="The host the server is running on", default="localhost")
+@click.option("--port", help="Port the server is running on", default=443)
+@click.option("--scheme", help="tcp/http/https", default="https")
+@click.option("--batch-size", help="How many samples to send", default=100000)
+@click.option("--parallelism", help="Maximum asynchronous jobs to run", default=10)
+@click.option(
+    "--metadata",
+    type=key_value_par,
+    multiple=True,
+    default=(),
+    help="Key-Value pair to be entered as metadata labels, may use this option multiple times. "
+    + "to be separated by a comma. ie: --metadata key,val --metadata some key,some value",
+)
+@click.option(
+    "--session-config",
+    type=yaml.safe_load,
+    default="{}",  # noqa:P103
+    help="Config json/yaml to set on the requests.Session object. Useful when needing to supply"
+    + "authentication parameters such as header keys. ie. --session-config {'headers': {'API-KEY': 'foo-bar'}}",
+)
+@click.pass_context
+def client_cli(ctx: click.Context, *args, **kwargs):
```

Comment:
I think `gordo_client` is a better name for this function. At least in constant with the command name

Reply:
fixed